Nodejs/Product/Nodejs/Snippets/1033/Nodejs/switch.snippet

```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<CodeSnippets
+    xmlns="http://schemas.microsoft.com/VisualStudio/2005/CodeSnippet">
+  <CodeSnippet Format="1.0.0">
+    <Header>
+      <Title>switch</Title>
+      <Description>Code Snippet for a Switch Statement</Description>
+      <Shortcut>switch</Shortcut>
+      <Author>Microsoft Corporation</Author>
+      <SnippetTypes>
+        <SnippetType>Expansion</SnippetType>
+      </SnippetTypes>
+    </Header>
+    <Snippet>
+      <Declarations>
+        <Literal>
+          <ID>switch_on</ID>
+          <Default>switch_on</Default>
+        </Literal>
+      </Declarations>
+      <Code Language="Nodejs">
+        <![CDATA[switch ($switch_on$) {
+    case:$end$

```

Comment:
I wouldn't bother including the ```case``` here and just end on the line in ```default```.

The reason for this is that we generally don't want the snippets to produce syntax errors, and in this case it would unless the user navigates to before the colon in case, types in their case statement, etc.